bin/merge_json_files.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
```

Comment:
This new script is used to merge the _qc.json files into the final `${id}.QC` json.